prime-router/src/main/resources/metadata/hl7_mapping/ORU_R01/ORU_R01-deidentified.yml

```diff
@@ -8,87 +8,135 @@ constants:
 elements:
   # PID 3.1
   - name: deidentified-patient-identifier
-    condition: '%patientPath.identifier[0].extension(%`rsext-cx-identifier`).extension.where(url = "CX.5").value != "PI"'
-    value: ['"<deidentified>"']
+    condition: '%patientPath.identifier[0].extension(%`rsext-cx-identifier`).extension.where(url = "CX.5").value !=
+     "PI" and %patientPath.identifier[0].extension(%`rsext-cx-identifier`).extension.where(url = "CX.5").value != 
+     "PT" and %patientPath.identifier[0].extension(%`rsext-cx-identifier`).extension.where(url = "CX.5").value != "SID"'
```

Comment:
I think this have to be OR'ed instead of AND'ed, or am I reading this wrong?

```
Remove if PID-3.5 does not equal "PI", "PT", or "SID"; else, no change
```

Reply:
I think it is saying it doesn't want it to equal any of them, so that would be and.